### PR TITLE
[bug 838125] Tell people what they are unsubscribing from

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: kitsune/tidings/templates/tidings/unsub.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{# vim: set ts=2 et sts=2 sw=2: #}
```
Comment (mythmon) [inline kitsune/tidings/templates/tidings/unsub.html]: Lets get rid of these.

**Comment by rehandalal** [comment]
done ^

File: kitsune/tidings/views.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    correct).
+
+    The templates assume use of the Jinja templating engine via jingo.Loader
+    and the presence of a ``base.html`` template containing a ``content``
+    block.
+
+    If you aren't using Jinja via jingo.Loader, you can replace the templates
+    with your own django templates.
+
+    """
```
Comment (mythmon) [inline kitsune/tidings/views.py]: :clap: :+1:

File: kitsune/tidings/templates/tidings/unsub.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      {% trans object=watch.content_object %}
+        You will be unsubscribed from:
+        <br><strong>{{ object }}</strong>
+      {% endtrans %}
+    </p>
+    <form action="" method="post">
+      {{ csrf() }}
+      <div class="submit-or-cancel">
+        <input type="submit" value="{{ _('Unsubscribe') }}" />
+        <a href="/"{# Discards l10n. Better ideas? #}>{{ _('Cancel') }}</a>
```
Comment (mythmon) [inline kitsune/tidings/templates/tidings/unsub.html]: Can we use something like `reverse('home')`? I think SUMO's url reverser does locale codes.

**Comment by mythmon** [comment]
I want to test this manually. Right now I'm importing a database, so I can't do much, but I'll try it out later.

**Comment by rehandalal** [comment]
fixed url ^

**Pull request merged**